zammad/templates/statefulset.yaml

```diff
@@ -237,12 +244,14 @@ spec:
               mountPath: /etc/nginx/sites-enabled/default
               subPath: default
               readOnly: true
-            - name: {{ include "zammad.fullname" . }}-tmp
-              mountPath: /tmp
```

Comment:
`/tmp` must also be present as a writable directory, right now nginx fails to start:
`nginx: [emerg] open() "/tmp/nginx.pid" failed (30: Read-only file system)`

Reply:
Thanks. Already fixed it :)

I also thought about to remove nginx from the container and use https://hub.docker.com/r/nginxinc/nginx-unprivileged instead, which should not have issues like that.

But that can be done later.

Reply:
Thank you.